packages/blockchain-api/package.json

```diff
@@ -16,6 +16,7 @@
     "deploy": "./deploy.sh"
   },
   "dependencies": {
+    "@celo/contractkit": "0.1.6",
```

Comment:
@annakaz this isn't available on npm so we should use 0.1.5

Reply:
0.1.6 has now been published, so leaving this version as is
https://www.npmjs.com/package/@celo/contractkit